doc/press-team.rst

```diff
@@ -23,16 +23,19 @@ Current team members
 - Vincent Breitmoser / valodim
 
 
-
-Accountability of press-team
------------------------------------------------------------
+Community interactions and responsibility
+-----------------------------------------
 
 If the press-team becomes aware of reporting, it will usually send
-a link of press articles to the community mailing list.  The press-team is
-accountable in that anyone can ask how media reporting came about and
-how and who, if anyone, was involved from the press-team.  Note that
-often journalists do not ask back before they publish articles, or do
-not even contact a project before reporting.
+a link of press articles to the community mailing list.  Anyone
+can ask how media reporting came about and how and who, if anyone, was
+involved from the press-team.  Note that often journalists do not ask
+back before they publish articles, or do not even contact a project
+before reporting.
+
+Press-team members remind each other when press-team internal
+discussions leak into community matters that are better discussed
```

Comment:
isn't it the opposite, i.e. "when community matters leak into press-team internal"?

Reply:
a bit ambigious but the remainder of the sentence clarifies it i think "that are better discussed through issues or mailing lists".  i  replaced "leak into" with "touch"  FWIW.